CONTRIBUTING.md

```diff
@@ -78,6 +78,46 @@ Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe
 
 * Discord - [#jest](https://discordapp.com/channels/102860784329052160/103622435865104384) on [Reactiflux](http://www.reactiflux.com/)
 
+## How to try a development build of Jest in another project
+
+To link `jest` on the command line to `jest-cli/bin/jest.js` in a development build:
+
+```sh
+cd /path/to/your/Jest_clone/packages/jest-cli
+npm link
+```
+
+To build Jest:
+
+```sh
+cd /path/to/your/Jest_clone
+
+# Do one of the following:
+
+# Check out a commit from another contributor, and then
+npm run build
+
+# Save your changes to Jest, and then
+npm test # which also builds Jest
+```
+
+To run tests in another project with the development build of Jest:
+
+```sh
+cd /path/to/another/project
+jest [options] # run jest-cli/bin/jest.js in the development build
+```
+
+* For more information about options, see `test` under `scripts` in the `package.json` file for that project.
+
+* You can still `npm test` to compare the installed version of Jest to the development build.
```

Comment:
Yea, but if your `"test"` npm is `"test": "jest"` it will take `jest` available through node path, which is a globally linked version to your dev build, right? 
I use `node node_modules/.bin/jest` to be sure I'm running local jest.